### PR TITLE
Update 11-licensing.md

### PR DESCRIPTION
**Pull request by HeatherNunn**
Defined acronym for clarity

### CODE REVIEW

File: _episodes/11-licensing.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ objectives:
 - "Choose a proper license."
 - "Explain differences in licensing and social expectations."
 keypoints:
-- "People who incorporate GPL'd software into their own software must make their software also open under the GPL license; most other open licenses do not require this."
+- "People who incorporate General Public Licenced (GPL'd) software into their own software must make their software also open under the GPL license; most other open licenses do not require this."
```
Comment (maxim-belkin) [inline _episodes/11-licensing.md]: I think you can omit `'d` now because it is no longer necessary.

**Comment by ErinBecker** [comment]
Thanks for this @HeatherNunn!

**Comment by maxim-belkin** [comment]
[witty comment about turnaround times]

**Comment by ErinBecker** [comment]
@maxim-belkin - Understood! We're currently onboarding new Maintainers. You should be seeing better turn around times soon!

**Pull request merged**